app/scenes/Collection.js

```diff
@@ -38,18 +48,31 @@ import { newDocumentUrl, collectionUrl } from "utils/routeHelpers";
 
 function CollectionScene() {
   const params = useParams();
+  const history = useHistory();
+  const match = useRouteMatch();
   const { t } = useTranslation();
   const { documents, policies, collections, ui } = useStores();
   const team = useCurrentTeam();
   const [isFetching, setFetching] = React.useState();
   const [error, setError] = React.useState();
   const [permissionsModalOpen, setPermissionsModalOpen] = React.useState(false);
 
-  const collectionId = params.id || "";
-  const collection = collections.get(collectionId);
-  const can = policies.abilities(collectionId || "");
+  const id = params.id || "";
+  const collection = collections.getByUrl(id);
```

Comment:
```suggestion
  const collection = collections.getByUrl(id) || collections.get(id);
```

Would it be useful for backwards compat and avoid the extra fetch call when loading an old-style url?